devito/ir/iet/efunc.py

```diff
@@ -86,8 +91,9 @@ def make_efunc(name, iet, dynamic_parameters=None, retval='void', prefix='static
     """
     Shortcut to create an ElementalFunction.
     """
-    return ElementalFunction(name, iet, retval, derive_parameters(iet), prefix,
-                             dynamic_parameters)
+    return ElementalFunction(name, iet, retval=retval,
+                             parameters=derive_parameters(iet), prefix=prefix,
```

Comment:
I would lift this out of the call, probably cleaner? `parameters=derive_parameters(iet),`